src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TimelineEventFactory.kt

```diff
@@ -35,6 +35,8 @@ class TimelineEventFactory {
    */
```

Comment:
Additionally, I could create events for each of the Goals added when a new ActionPlan is created (within the `actionPlanCreatedEvent()` method which isn't shown here). They could all have the same correlationId and therefore be grouped accordingly.

Reply:
Done.